zallet/src/cli.rs

```diff
@@ -128,3 +132,15 @@ pub(crate) struct GenerateMnemonicCmd {}
 #[derive(Debug, Parser)]
 #[cfg_attr(outside_buildscript, derive(Command))]
 pub(crate) struct ImportMnemonicCmd {}
+
+/// `rpc` subcommand
+#[cfg(feature = "rpc-cli")]
+#[derive(Debug, Parser)]
+#[cfg_attr(outside_buildscript, derive(Command))]
+pub(crate) struct RpcCliCmd {
+    /// The JSON-RPC command to send to Zallet.
+    pub(crate) command: String,
+
+    /// Any parameters for the command.
+    pub(crate) params: Vec<String>,
```

Comment:
I'd like to support both array params and map params:
```
$ zallet rpc z_gettotalbalance null true
$ zallet rpc z_gettotalbalance --include_watch_only=true
```
but I don't believe Clap supports arbitrary unnamed non-positional args, and I don't have time to write our own parser to implement it.